src/test/recipe.spec.ts

```diff
@@ -0,0 +1,509 @@
+import {
+  chai,
+  expect,
+  app,
+  StatusCodes,
+  RecipeDocument,
+} from "./config/server.config";
+import {
+  recipes,
+  deleteAllRecipes,
+  addSomeRecipes,
+} from "./mocks/recipe.mocks";
+import { getToken } from "./mocks/user.mocks";
+
+let token = "Bearer ";
+let _id: string;
+let updateRecipeId: string;
+
+before("Add user and get token", async function () {
+  token += await getToken();
+  _id = await addSomeRecipes();
+});
+
+after("Delete all recipes", async function () {
+  await deleteAllRecipes();
```

Comment:
I would flush whole database, not only recipes. And I think I would do this in beforeEach. With afterEach approach you have to make sure all other .spec.ts file has it properly implemented. With beforeEach we do not have to care about other files. Also beforeEach, not before, because we want the database to be in init state before each and every test